src/renderer/routes/test_sequencer_panel/components/modals/SequencerGalleryModal.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/renderer/components/ui/dialog";
+import { LayoutGrid } from "lucide-react";
+import { ScrollArea } from "@/renderer/components/ui/scroll-area";
+import { Separator } from "@/renderer/components/ui/separator";
+import { Button } from "@/renderer/components/ui/button";
+import { useImportAllSequencesInFolder } from "@/renderer/hooks/useTestSequencerProject";
+
+type AppGalleryModalProps = {
+  isGalleryOpen: boolean;
+  setIsGalleryOpen: (open: boolean) => void;
+};
+
+export const SequencerGalleryModal = ({
+  isGalleryOpen,
+  setIsGalleryOpen,
+}: AppGalleryModalProps) => {
+  const setOpen = () => {
+    setIsGalleryOpen(true);
+  };
+
+  const useImport = useImportAllSequencesInFolder();
+
+  const handleSequenceLoad = (BaseFolderName: string) => {
+    const relativePath = `src/renderer/data/apps/sequencer/${BaseFolderName}/`;
```

Comment:
I don't think this is going to work when we package the app tho, this directory certainly will not exist

Reply:
Oh, facts I forgot to test it by packaging the app, it's not the only place in the App using relative path to open the examples. But the packaging might by done with `renderer` only

Reply:
Oh never mind, the other place just use the raw text

Reply:
Switched to the same flow as the Test profile. (I'll need to add a "download" fall back if git is not install this week)

Reply:
Revert to local